### DISCUSSION TITLE
We had an online meetup! (Wednesday 3rd 20:00 GMT)

### DISCUSSION THREAD

**Discussion by chrisjsewell** [sphinx-doc/sphinx#12443] [Community Announcements]
edit: Planned Wednesday 3rd 20:00 GMT  https://us04web.zoom.us/j/76159223710?pwd=rBkvaaf6ia7WfdvSHtNWalqQ8rJcLD.1

---

Hey all, I would like to see if we can have a little meet up and put some names to faces!

Here is a doodle with some proposed times: https://doodle.com/meeting/participate/id/ax74pkle

I'm European centric, but have tried to provide a broad range of times 😅

This meeting is not intended to be anything too formal, just saying hi and putting some faces to names,
and seeing how many people are interested and how it might work.
But my hope is that it may lead to a more scheduled meeting every ~few months

**Comment by chrisjsewell** [comment]
Perhaps though if you want to suggest any discussion topics, add them here!

**Reply by chrisjsewell** [reply]
Note: I would envisage the primary audience would be predominantly people interested in the development of "sphinx-core": people specifically contributing to sphinx core, but then also those writing extensions for sphinx that may obviously have an interest in how sphinx is developing. 
Perhaps there would also be space for one or more extension developers / users to "show-case" their extension and share their experience.

**Reply by jdillard** [reply]
I fall into both camps, interested in sphinx-core, but also participate and enjoy the extension community. I think there is also a third camp that could be users, possibly those that want to be more of a "power user".

For the first meeting it might be useful to do intros and cover things like how you got into Sphinx, what things you have built with it, your area of interest when it comes to participating, etc.

**Comment by Carreau** [comment]
Did you ended up selecting a date and time ? I might end up joining depending on which but don't want to promise so I haven't voted. I'd still like to know if/when.

**Comment by chrisjsewell** [comment]
Hey all, ok sorry for the late confirmation, but Wednesday 3rd 20:00 GMT (tomorrow) was the slot most voted for, so I chose that 😅

https://us04web.zoom.us/j/76159223710?pwd=rBkvaaf6ia7WfdvSHtNWalqQ8rJcLD.1

Perhaps like this comment if you intend to participate.

cc @jdillard @wlach @picnixz @AA-Turner @birkenfeld @danwos @shimizukawa @jayaddison @timhoffm @melissawm @trallard @kaycebasques

**Reply by Carreau** [reply]
Thanks, I will try to make it.

**Reply by picnixz** [reply]
I'm sorry, but I'm defending my thesis on July 4th. I would have been interested to join otherwise!

**Reply by trallard** [reply]
Thanks for the ping! Will try and make it.

**Reply by birkenfeld** [reply]
I'm still recovering from an encounter with Covid, so unfortunately I won't join either :(

**Reply by melissawm** [reply]
I will be there!

**Reply by chrisjsewell** [reply]
> I'm sorry, but I'm defending my thesis on July 4th. I would have been interested to join otherwise!

good luck!

**Reply by chrisjsewell** [reply]
> I'm still recovering from an encounter with Covid, so unfortunately I won't join either :(

Oh damn, no problem, get well soon!

**Reply by jayaddison** [reply]
I won't be joining tomorrow but thanks for the invitation!

**Reply by timhoffm** [reply]
Thanks, I will try to make it.

**Comment by chrisjsewell** [comment]
Thanks to everyone that joined the meeting!

I have some notes jotted down, and will maybe try to write them down here tomorrow 

and yes a few follow-ups noted down, in-particular talking with the scipy folks, on possible contributions and user survey

**Reply by jdsalaro** [reply]
@chrisjsewell , minor request for future meet-ups as I'd be interested in taking part:
would it be possible to share the agreed upon date as a new comment and not an edit or, alternatively, directly to the Sphinx mailing list?
By the time I got a notification regarding @Carreau's inquiry it was unfortunately too late for me to move plans I had for the 3rd.
Thank you for the notes (looking forward to checking them out)!

**Reply by trallard** [reply]
Sorry I missed it, hopefully next time I will be there!

**Comment by Carreau** [comment]
Sorry to hijack the thread, @chrisjsewell would you mind opening an annoucenement "8.0rc1 has been published" and susggest people comment if they have tried it with their project and all is well.

**Reply by AA-Turner** [reply]
Whilst I am not Chris, see #12682.

A

**Comment by AA-Turner** [comment]
Autumn 2025 edition: #14044.

A
